Dockerfile.fortran

```diff
@@ -0,0 +1,109 @@
+# on macOS this image requires rosetta 2 support for virtualization
+# the intel compilers do not support the arm architecture so you must build
+# on an amd platform, or virtualize
+# FROM --platform=linux/amd64 fedora:35
+FROM fedora:35
+
+RUN dnf -y update \
+    && dnf -y install \
+        cmake \
+        gcc-c++ \
+        gcc-gfortran \
+        # gdb \
+        git \
+        lcov \
+        make \
+        netcdf-fortran-devel \
+        # wget \
+        # glibc-devel \
+        valgrind \
+    && dnf clean all
+
+# Install json-fortran
+RUN curl -LO https://github.com/jacobwilliams/json-fortran/archive/8.2.0.tar.gz \
+    && tar -zxvf 8.2.0.tar.gz \
+    && cd json-fortran-8.2.0 \
+    && mkdir build \
+    && cd build \
+    && cmake -D SKIP_DOC_GEN:BOOL=TRUE .. \
+    && make install
+
+# Set environment variables
+ENV FC=gfortran
+ENV JSON_FORTRAN_HOME="/usr/local/jsonfortran-gnu-8.2.0"
+
+# Copy the musica core code
+COPY . musica
+
+# Build
+RUN cd musica/musica \
+    && cmake -S. \
+             -Bbuild \
+             -D ENABLE_TESTS=ON \
+    && cd build \
+    && make install -j 8    
+
+# Build
+RUN cd musica/musica-fortran \
+    && cmake -S . \
+             -Bbuild \
+             -D ENABLE_TESTS=ON \
+    && cd build \
+    && make
+
+######
+######
+# TODO: jiwon 8/4 - We need these but it takes too long time to build so I will skip for now
```

Comment:
We can copy the micm file for intel https://github.com/NCAR/micm/blob/main/Dockerfile.intel. I didn't realize when I originally wrote this that intel published docker images